src/CSnakes.Tests/GeneratedSignatureTests.cs

```diff
@@ -63,6 +63,32 @@ public class GeneratedSignatureTests
     [InlineData("def hello(a: str, b: int = 4, *, kw: str) -> None:\n ...\n", "void Hello(string a, string kw, long b = 4)")]
     [InlineData("def hello() -> str | int: \n  ...\n", "PyObject Hello()")]
     [InlineData("def escape(s: str, quote: bool = True) -> str: ...\n", "string Escape(string s, bool quote = true)")]
+    [InlineData("""
+        
+        # csharp: ignore
+        def foo(bar):
+            pass
+        
+        """, "PyObject Foo(PyObject bar)",
+        Label = "C# ignore comment on line above")]
+    [InlineData("""
+        
+        def foo(
+            bar  # csharp: ignore
+        ):
+            pass
+        
+        """,
+        "PyObject Foo(PyObject bar)",
+        Label = "C# ignore comment in parameter list")]
+    [InlineData("""
+        
+        def foo(bar):
+            # csharp: ignore
+            pass
+        
+        """, "PyObject Foo(PyObject bar)",
```

Comment:
nit: for consistency:

```suggestion
        """,
        "PyObject Foo(PyObject bar)",
```